app/models/user_export.rb

```diff
@@ -14,6 +15,7 @@ def ignore_missing_post_uploads
 
   def self.remove_old_exports
     UserExport.where('created_at < ?', 2.days.ago).find_each do |user_export|
+      Post.where(topic_id: user_export.topic_id).find_each { |p| p.destroy! }
```

Comment:
We probably want to run this in a transaction. 

Also, what happens if an error is raised while destroying a user_export? Do we want to raise the error and stop the clean up midway or should we rescue errors and log?

Reply:
Wrapped in transaction and rescued + logged errors.